updatecli/updatecli.d/trusted.ci.sp.endate.yaml

```diff
@@ -0,0 +1,62 @@
+name: "Generate new end date for trusted.ci.jenkins.io controller service principal"
```

Comment:
```suggestion
name: "Generate new end date for the trusted.ci.jenkins.io controller Azure AD Application password"
```

Reply:
It's an Azure AD password, not a Service Principal password as per https://github.com/jenkins-infra/shared-tools/blob/adae7d43808f77c2a94bac3846d6219da93b3261/terraform/modules/azure-jenkinsinfra-controller/main.tf#L303-L307